lib/cog_api/http/client.ex

```diff
@@ -134,20 +134,36 @@ defmodule CogApi.HTTP.Client do
     RelayGroups.delete(relay_group_id, endpoint)
   end
 
-  def relay_group_add_relay(relay_group_id, relay_id, %Endpoint{}=endpoint) do
-    RelayGroups.update_memberships(:add, relay_group_id, relay_id, endpoint)
+  def relay_group_add_relays_by_name(relay_group_name, relay_names, %Endpoint{}=endpoint) do
```

Comment:
@lhaskins there are a few reasons:

* One is that you have to know more about how the function is written in order to call it. So for example you have to know that you can either call the function passing a relay group id and a list of relay/bundle ids or if you want to pass relay group and relay/bundle names you need to create a map and insert the values in the proper field. The only reason for the map is to match the proper function head, it really has no bearing on the data itself.
* It also makes reasoning about the code easier. You can quickly look at the function definition and understand both what the expected data should look like and how that data will flow through the function.
* It just makes for a more clear statement of intent when you are calling the function. It's easy to look at the code and determine what it's supposed to do.